src/test/resources/json/getconsignment_data_all.json

```diff
@@ -105,7 +105,7 @@
       "parentFolder": "ALL CONSIGNMENT DATA PARENT FOLDER",
       "fileChecks": {
         "antivirusProgress": {
-          "filesProcessed": 1
+          "filesProcessed": 2
```

Comment:
Why has this changed? IIRC, it was deliberately checking a different value to `checksumProgress`.